packages/@aws-cdk-testing/framework-integ/test/aws-stepfunctions-tasks/test/emr/integ.emr-create-cluster-add-step-runtime-role.ts

```diff
@@ -0,0 +1,115 @@
+import * as sfn from 'aws-cdk-lib/aws-stepfunctions';
+import * as iam from 'aws-cdk-lib/aws-iam';
+import * as tasks from 'aws-cdk-lib/aws-stepfunctions-tasks';
+import { App, Stack } from 'aws-cdk-lib';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+
+/*
+ * Create a state machine with an EMR cluster and adds a step that uses a runtime role.
+ *
+ * Prerequisites:
+ * 1. Fill in the instances block for EmrCreateCluster
+ * 2. Create the EMR security configuration, see https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-create-security-configuration.html and
+ *    https://aws.amazon.com/blogs/big-data/introducing-runtime-roles-for-amazon-emr-steps-use-iam-roles-and-aws-lake-formation-for-access-control-with-amazon-emr/
+ * 3. Add the security configuration to EmrCreateCluster
+ *
+ * Stack verification steps:
+ * The generated State Machine can be executed from the CLI (or Step Functions console)
+ * and runs with an execution status of `Succeeded`.
+ *
+ * -- aws stepfunctions start-execution --state-machine-arn <state-machine-arn-from-output> provides execution arn
+ * -- aws stepfunctions describe-execution --execution-arn <state-machine-arn-from-output> returns a status of `Succeeded`
+ */
+
+const app = new App();
+const stack = new Stack(app, 'aws-cdk-emr-add-step-runtime-role');
+
+const createClusterStep = new tasks.EmrCreateCluster(stack, 'EmrCreateCluster', {
+  instances: {
+    /*
+    instanceFleets: [
+      {
+        instanceFleetType: tasks.EmrCreateCluster.InstanceRoleType.MASTER,
+        instanceTypeConfigs: [
+          {
+            instanceType: 'm5.xlarge',
+          },
+        ],
+        targetOnDemandCapacity: 1,
+      },
+    ],
+    ec2SubnetId: 'subnet-xxxxxx',
+    */
+  },
```

Comment:
```suggestion
const vpc = new ec2.Vpc(stack, 'Vpc', { restrictDefaultSecurityGroup: false });

const createClusterStep = new tasks.EmrCreateCluster(stack, 'EmrCreateCluster', {
  instances: {
    instanceFleets: [
      {
        instanceFleetType: tasks.EmrCreateCluster.InstanceRoleType.MASTER,
        instanceTypeConfigs: [
          {
            instanceType: 'm5.xlarge',
          },
        ],
        targetOnDemandCapacity: 1,
      },
    ],
    ec2SubnetId: vpc.publicSubnets[0].subnetId,
  },
```
Let's provide a comprehensive example, integration tests can be a useful template for users who want to implement the functionality (may need adjustments).
Creating a cluster from the console requires a VPC and subnet(s), I wasn't able to find documentation about it.